### PR TITLE
Kendo QoL and Fixes

### PR DESCRIPTION
**Pull request by AwkwardDryad**
[QOL][BUGFIX]

## About the PR 
- Intents now update properly when pulling a shinai out of the bag
- Added block properties to shinai: they can be used to mitigate damage from non-shinai weapons now (including katanas and...ahem...csabers...So far this seems really nicely balanced with other facets of the game at the moment, so if it ends up being a problem, it can be toned down a little later)
- Added a kendo crate to QM
- Added kendo gear to the hidden workshop
- Made a super hacky temporary fix to make the descriptions of kendo items look slightly less zalgo (Specifically, the issue is that if the first character of the description isn't a basic a-z character, it kinda gets potatoed, so if anyone has a proper fix for that, please let me know :heart: )

## Why's this needed? 
- balance, functionality, followup content, and bugfixes :D

## Changelog

```
(u)AwkwardDryad:
(+)Shinai can now be used to mitigate damage from other weapons while blocking
(+)Additional kendo gear is available from QM
(+)Kendo gear now spawns in the hidden workshop
```

### CODE REVIEW

File: code/obj/item/kendo.dm [hunk 1]
```diff
@@ -147,16 +151,17 @@
...
-                    attacker.show_text("<b>You knock the [I] right out of [defender]'s hands!</b>","green")
+                    return //stops damage if parried or blocked, if not, itll check for a disarm
+
+            if((attacker.a_intent=="disarm") && prob(20) && defender.equipped())
+                var/obj/item/I = defender.equipped()
+                defender.u_equip(I)
+                I.set_loc(defender.loc)
+                var/target_turf = get_offset_target_turf(I.loc,rand(5)-rand(5),rand(5)-rand(5))
+                SPAWN_DBG(1 DECI SECOND)
+                I.throw_at(target_turf,3,1)
```
Comment (Tarmunora) [inline code/obj/item/kendo.dm:162]: missing indent after spawn?

Comment (AwkwardDryad) [reply code/obj/item/kendo.dm:162]: I thought that might be an issue, but ill fix it soon

Comment (AwkwardDryad) [reply code/obj/item/kendo.dm:161]: fixy fixed～

Comment (Tarmunora) [reply code/obj/item/kendo.dm:161]: Don't you *want* the throw_at to be in a spawn though? Otherwise it will hold up the proc (throw_at sleeps)

Comment (AwkwardDryad) [reply code/obj/item/kendo.dm]: true...i readded it :P

**Pull request merged**